### PR TITLE
Add note about edX bulk email service vs. MailChimp

### PR DESCRIPTION
**Pull request by srpearce**
Ticket: [DOC-931](https://openedx.atlassian.net/browse/DOC-931)

@mhoeber or @lamagnifica Please review.

I didn't update the change logs here because I plan to work on several more tickets today. I'll create a separate PR to update the change logs for the ones that get merged.

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/running_course/bulk_email.rst [hunk 1]
```diff
@@ -4,12 +4,18 @@
...
+styling, and can include links to videos, social media pages for the course,
+and other material. All course contributors who are assigned the Staff or
+Admin role can use bulk emails to communicate with course participants before,
+during, and after the course run.
+
+.. note:: 
+  Some courses use services such as MailChimp to send bulk email. EdX
+  recommends that you use either the edX bulk email service or a different
+  service, but not both. If you use more than one service to send email, your
+  messages may be marked as spam, and learners may not see them.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/bulk_email.rst]: messages may be marked as spam --> messages are more likely to be marked as spam
learners may  --> learners might

File: en_us/shared/building_and_running_chapters/running_course/bulk_email.rst [hunk 2]
```diff
@@ -4,12 +4,18 @@
...
-after the course run. 
+For courses on edx.org, you can send bulk email messages to course
+participants directly from the Instructor Dashboard. Messages can use HTML
+styling, and can include links to videos, social media pages for the course,
+and other material. All course contributors who are assigned the Staff or
+Admin role can use bulk emails to communicate with course participants before,
+during, and after the course run.
+
+.. note:: 
+  Some courses use services such as MailChimp to send bulk email. EdX
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/bulk_email.rst]: ...use ^third party services... (?)

**Comment by lamagnifica** [comment]
:+1: a few suggestions. I'd also suggest asking Sarina to take a look (but this is sort of the wrong week for that ask).

**Comment by srpearce** [comment]
@sarina When you have time, could you take a quick look at this? It's from [DOC-931](https://openedx.atlassian.net/browse/DOC-931), which requests that we add a note telling course teams to use either edX's bulk email system _or_ a third-party system like MailChimp, but not both.

File: en_us/shared/building_and_running_chapters/running_course/bulk_email.rst [hunk 3]
```diff
@@ -4,12 +4,19 @@
...
+For courses on edx.org, you can send bulk email messages to course
+participants directly from the Instructor Dashboard. Messages can use HTML
+styling, and can include links to videos, social media pages for the course,
+and other material. All course contributors who are assigned the Staff or
+Admin role can use bulk emails to communicate with course participants before,
+during, and after the course run.
+
+.. note:: 
+  Some courses use third party services such as MailChimp to send bulk email.
+  EdX recommends that you use either the edX bulk email service or a different
```
Comment (sarina) [inline en_us/shared/building_and_running_chapters/running_course/bulk_email.rst]: Is "recommends" the strongest way we can word this? The worst case scenario is actually that _all_ edX mail gets marked as spam, or the user quits using edX for sending too many emails. The truth of the matter is, "only use one service to email your students. Do Not use the bulk emailer in conjunction with other services."

**Pull request merged**